state-sync/state-sync-v1/src/coordinator.rs

```diff
@@ -269,8 +270,8 @@ impl<T: ExecutorProxyTrait, M: MempoolNotificationSender> StateSyncCoordinator<T
                 // Time request handling
                 let _timer = counters::PROCESS_MSG_LATENCY
                     .with_label_values(&[
-                        &peer.network_id().to_string(),
-                        &peer.peer_id().to_string(),
+                        peer.network_id().as_str(),
+                        peer.peer_id().short_str().as_str(),
```

Comment:
What's the reason for using the hex string here? Is this something all applications should do (e.g., for debugging purposes?)

Reply:
It shortens the metric names, honestly it doesn't really matter, but mempool was doing it and I was changing one of them for consistency.

It should however at least do the as_str().  Really up to you as you're the owner of state sync, I think the names are nicer shorter given the extremely low chance of a name collision.

Reply:
both `peer.peer_id().to_string()` and `peer.peer_id().short_str()` return a hex string, the `short_str` just truncates it to  8 characters (like "1234feba") and does it without any allocation. personally, I find it much more difficult to read the longer 32-character version, especially when reading through many logs / metrics etc...